deck-builder/main.js

```diff
@@ -1272,10 +251,134 @@ $cardList.addEventListener('click', function (event) {
       class: event.target.closest('tr').className
     }
   }
-  addCard($card)
+  var fullCardInfo = addCard($card)
   var deckCount = deckCounter()
   $characterCounter.textContent = deckCount.characterCount
   $eventCounter.textContent = deckCount.eventCount
   $climaxCounter.textContent = deckCount.climaxCount
   $cardCounter.textContent = deckCount.cardCount + '/50'
+
+  if (fullCardInfo !== null) {
```

Comment:
Try to flatten your conditional logic by returning early. Here is another large `if` with no else.